python/openai/openai_frontend/main.py

```diff
@@ -28,12 +28,24 @@
 
 import argparse
 import signal
+import sys
 from functools import partial
 
 import tritonserver
 from engine.triton_engine import TritonLLMEngine
 from frontend.fastapi_frontend import FastApiFrontend
 
+try:
+    from tritonfrontend import KServeGrpc, KServeHttp
+except:
+    print(
+        """
+          tritonfrontend is not present in this environment.
+          Check /opt/tritonserver/python for tritonfrontend*.whl/
+          If present, please pip install.
+          """
```

Comment:
This prints with some weird spacing. Can we print something like this?
```python
print(
    "[WARNING] The 'tritonfrontend' package was not found. "
    "KServe frontends won't be available through this application without it. "
    "Check /opt/tritonserver/python for tritonfrontend*.whl and pip install it if present."
)
```

Reply:
New output formatting:
![image](https://github.com/user-attachments/assets/b537ab5a-e74b-4e8c-94bf-5cb627d6a434)
Thank you for suggestion!